notebooks/experiments/conditional_diffusion/dna_diff_baseline_conditional_UNET.ipynb

```diff
@@ -0,0 +1,7119 @@
+{
```

Comment:
<p>Can we also do Garbage Collection and Empty cache after each step through the dataloader viz.</p><p> </p><pre class="ql-syntax" spellcheck="false">for epoch in range(...):
  for idx, sample in enumerate(dataloader):
  ...
  # ⭐️⭐️ Garbage Collection
  torch.cuda.empty_cache()
  _ = gc.collect()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinellolab/DNA-Diffusion/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>